shared/src/commonTest/kotlin/vn/luongvo/kmm/survey/data/repository/AuthRepositoryTest.kt

```diff
@@ -168,8 +168,8 @@ class AuthRepositoryTest {
     }
 
     @Test
-    fun `when calling clearToken - it executes the local data source`() = runTest {
-        repository.clearToken()
+    fun `when calling clearLocalToken - it executes clearToken in the local data source`() = runTest {
```

Comment:
```suggestion
    fun `when calling clearLocalToken - it executes clear method in the local data source`() = runTest {
```

Reply:
Fixed in https://github.com/luongvo/kmm-survey/pull/72/commits/4a4cd5fbdaf4703e1b81b3bc44135261ffd579fa